src/pages/contributing/components.mdx

```diff
@@ -5,89 +5,28 @@ description: Component contributions can take several forms. Most are either com
 
 <PageDescription>
 
-Component contributions can take different forms. Some are enhancements to existing components
-or a brand new component. It can even be a layout component which brings several components together to form a commonly used element, like a call to action button on a Lead Space.
+Component contributions can take different forms. Most contributions are enhancements to an existing component
+or a brand new component.
 
 </PageDescription>
 
 <AnchorLinks>
-  <AnchorLink>First steps to contributing</AnchorLink>
-  <AnchorLink>Component contribution</AnchorLink>
-  <AnchorLink>Documentation requirements</AnchorLink>
-  <AnchorLink>How to publish component usage documentation</AnchorLink>
+  <AnchorLink>How to write component guidance</AnchorLink>
+  <AnchorLink>Parts of a component contribution</AnchorLink>
+  <AnchorLink>Steps to contributing</AnchorLink>
 </AnchorLinks>
 
-## First steps to contributing
+## How to write component guidance
 
-To contribute a component to Carbon for IBM.com, start by opening a <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/new/choose" target="_blank">GitHub issue</a> and select Feature Request.
-
-Include a detailed description in which you:
-
-- Explain the rationale for the request
-- Detail the intent and/or usage
-- Clarify whether it's a variation of an existing component, or a new component
-- Include mockups of any fidelity (optional)
-- Include any inspirations from other products (optional)
-
-## Component contribution
-
-Component contributions ideally include all of the following parts.
-
-#### 1. A rationale
-
-Explain how your component will add value to the system. Carbon for IBM.com serves the
-widest possible range of IBM.com needs, and contributions that increase the scope of
-the system are more likely to be accepted. Be sure to include any user
-experience and interaction descriptions.
-
-#### 2. A design spec
-
-Create sizing and styling annotations for all aspects of the component. This
-spec should provide a developer with everything they need to create the design
-in code. Check out Carbon's <a href="https://github.com/carbon-design-system/carbon-website/wiki/Production-guidelines#spec-guidelines" target="_blank">production guidelines</a> to get started.
-
-You should include color tokens and type tokens used.
-
-<Row>
-<Column colLg={8}>
-
-![Example of a design spec](../../images/contributions/content-switcher-style-1.jpg)
-
-<Caption>Example of a design spec</Caption>
-
-</Column>
-</Row>
```

Comment:
I think there is a place for this content. **Contributing a component** is what AEM has been doing so documenting the official process would be good. We need to tell people, if they want a change/new component done to library, they need to 1. make a feature request in github; 2. justify the need; 3. give proposed design & spec; 4. contribute usage guidance. 

This PR's goal is add the component documentation templates. I think we can preserve some of these deleted content for now until a better version is written.

Reply:
@shixiedesign I didn't remove the content. I just changed the order of the topics.

Reply:
Oh! Of course 👍